client/components/info-popover/README.md

```diff
@@ -18,10 +18,57 @@ The `position` property can be one of the following values:
 - `left`
 - `right`
 
-### `InfoPopover` Usage
+#### `className`
+
+The `className` lets you specify the style class that the element should have.
+
+#### `gaEventCategory` 
+
+The `gaEventCategory` lets you specify the Google Analyics Category that you want the toggle event to have. 
+Also reqires the `popoverName` attribute. 
+
+#### `popoverName` 
+
+The `popoverName` lets you specify the Google Analyics Event name that you want the toggle event to have. 
+Also reqires the `gaEventCategory` attribute. 
+
+Turns into this even when opened: 
+analytics.ga.recordEvent( gaEventCategory, 'InfoPopover: ' + popoverName + 'Opened' );
+
+#### `ignoreContext`
+
+The `ignoreContext` lets you specify an component that you want to be on the inside clickOutside context. 
+So a context that you want to ignore. In most cases this is not needed but if you want to also have a label 
+that can trigger the opening and closing of the InfoPopover then you need to pass in the label component as a reference.
+
+### Basic `InfoPopover` Usage
 
 ```js
 <InfoPopover position="bottom left">
     This is some informational text
 </InfoPopover>
 ```
+
+
+```js
+
+handleAction( event ) {	
+	this.refs && this.refs.infoPop._onClick( event );
+},
+
+render() {
+	return (
+	<label onClick={ this.handleAction } ref="moreInfoLabel">More Info</label>
```

Comment:
I think this should have one more tab on the left, and also both <label> and <InfoPopover> should be inside of a container tag (<div>?), because returning two nodes makes baby react cry